tests/cloner_test.go

```diff
@@ -282,11 +282,20 @@ var _ = Describe("all clone tests", func() {
 			DescribeTable("[test_id:1355]Should clone data across different namespaces", func(targetSize string) {
 				pvcDef := utils.NewPVCDefinition(sourcePVCName, "1Gi", nil, nil)
 				pvcDef.Namespace = f.Namespace.Name
+
+				pvcDef.Labels = map[string]string{"test-label-1": "test-label-key-1"}
+				pvcDef.Annotations = map[string]string{"test-annotation-1": "test-annotation-key-1"}
+
```

Comment:
You should add labels and annotations to *DV*, not to PVC, and check they are passed to the *PVC*.

Reply:
I had a few issues unrelated to my code that Alex and I couldn't resolve..
I did a rebase and got conflicts in places I didn't edit, in addition to some errors I couldn't solve when I ran test-functional.
I tried to push anyway